src/test/scala/org/broadinstitute/dsde/workbench/sam/service/PolicyEvaluatorServiceSpec.scala

```diff
@@ -176,7 +177,7 @@ class PolicyEvaluatorServiceSpec extends AsyncFlatSpec with Matchers with TestSu
       r shouldBe false
     }
 
-    res.unsafeToFuture()
+    res.unsafeRunSync()
```

Comment:
Can you use your new `runAndWait` here?  and in each of the other tests that are still using `unsafeRunSync`?

Reply:
I think calling `unsafeRunSync` directly is better